plugins/package-managers/python/src/test/kotlin/PoetryTest.kt

```diff
@@ -25,7 +25,7 @@ import io.kotest.matchers.file.aFile
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
 
-import org.ossreviewtoolkit.plugins.packagemanagers.python.Pip.Companion.PYPROJECT_FILENAME
+import org.ossreviewtoolkit.plugins.packagemanagers.python.Poetry.Companion.PYPROJECT_FILENAME
```

Comment:
Semi-related: Should this apply the same `import`-simplification as in the previous commit?

Reply:
I'm not following. In which code location can the same simplification be applied?

Reply:
Nevermind, I somehow was mentally reading the diff of the first commit inverse, thinking that you've removed the import in favor of scoped access to the constant.